tools/lkl/Makefile

```diff
@@ -69,7 +70,7 @@ ifneq (,$(filter $(OUTPUT_FORMAT),elf64-x86-64 elf32-i386 elf64-x86-64-freebsd e
   # Virtual Distributed Ethernet configuration
   ifeq ($(vde),yes)
     export CONFIG_AUTO_LKL_VIRTIO_NET_VDE=y
-    CFLAGS     += -DCONFIG_AUTO_LKL_VIRTIO_NET_VDE
+    CFLAGS     += -DCONFIG_AUTO_LKL_VIRTIO_NET_VDE $(shell pkg-config --cflags vdeplug)
```

Comment:
I think CFLAGS is already updated in lib/Kbuild? Maybe you can pick up this commit:

https://github.com/tavip/linux/commit/20ed06560794f99fdcb6beaf0c764d60f1ecd0b5
 
and since it fixes errors when vde is not installed on the host and removes ifdef from the test.

Reply:
This portion tries to do the same with your commit (tavip/linux@20ed065), indeed.
and much complete solution including net-test fix, etc.

I will pull (cherry-pick) your commit in this PR.